application/listeners/listeners.go

```diff
@@ -76,6 +85,44 @@ func userCreated(e events.Event) {
 	domain.Logger.Printf("%s User Created ... %s", domain.GetCurrentTime(), e.Message)
 }
 
+func sendNewMessageNotification(e events.Event) {
+	if e.Kind != domain.EventApiMessageCreated {
+		return
+	}
+	domain.Logger.Printf("%s Message Created ... %s", domain.GetCurrentTime(), e.Message)
+
```

Comment:
If you make the change in the message.Create() function that I described, you'll need to do something like this here ...
```
  mEData := e.Payload["eventData"].(models.MessageCreatedEventData)

  data := map[string]interface{} {
    "postTitle": mEData.PostTitle,
    ...
```
  }